presto-hive/src/main/java/com/facebook/presto/hive/HivePartialAggregationPushdown.java

```diff
@@ -188,29 +192,29 @@ private boolean isAggregationPushdownSupported(AggregationNode partialAggregatio
             return true;
         }
 
-        private boolean isSupportedOrcTypeForMinMax(Type type)
+        private boolean isNotSupportedOrcTypeForMinMax(Type type)
         {
-            return !BOOLEAN.equals(type) &&
-                    type.getJavaType() != boolean.class &&
-                    !isRowType(type) &&
-                    !isArrayType(type) &&
-                    !isMapType(type) &&
-                    !TINYINT.equals(type) &&
-                    !VARBINARY.equals(type) &&
-                    !TIMESTAMP.equals(type) &&
-                    ((!VARBINARY.equals(type) && !VARCHAR.equals(type)) ||
-                            isPartialAggregationPushdownForVariableLengthDatatypesEnabled(session));
+            return BOOLEAN.equals(type) ||
+                    type.getJavaType() == boolean.class ||
+                    isRowType(type) ||
+                    isArrayType(type) ||
+                    isMapType(type) ||
+                    TINYINT.equals(type) ||
```

Comment:
Is adding tinyint (byte) correct? All other int types are not listed.

Reply:
All of these checks are just refactoring from the original code. See https://github.com/prestodb/presto/blob/release-0.280/presto-hive/src/main/java/com/facebook/presto/hive/HivePartialAggregationPushdown.java#L185